migrate.py

```diff
@@ -29,6 +30,53 @@
 PLUGIN_EXCEPTION_PATHS = {'modules': 'lib/ansible/modules', 'module_utils': 'lib/ansible/module_utils'}
 
 
+def _run_command(cmd=None, check_rc=True):
+    logger.debug(cmd)
+    if not isinstance(cmd, bytes):
+        cmd = cmd.encode('utf-8')
+    p = subprocess.Popen(
```

Comment:
Maybe just check_run/call?

Reply:
I didn't want to change run_command functions much in this PR. I'd prefer to do that separately.

Reply:
agreed, lets concentrate on functional changes needed to get this working

Reply:
Ah, I initially thought that you're adding this function, not moving it.